taskboot/build.py

```diff
@@ -147,9 +146,7 @@ def build_compose(target, args):
 
         # Write the produced image
         if output:
-            for tag in tags:
-                output_file = '{}.tar'.format(tag.replace(os.sep, "_").replace(":", "_"))
-                docker.save(tag, os.path.join(output, output_file))
+            docker.save(tags, os.path.join(output, '{}.tar'.format(service)))
```

Comment:
This used to be `name` before https://github.com/mozilla/task-boot/pull/38, maybe we want to keep that?

Reply:
:man_facepalming: